.github/workflows/ci.yml

```diff
@@ -26,4 +26,17 @@ jobs:
         docker run -d -p 50051:50051 casbin/casbin-server
       
     - name: Go Test
-      run: go test ./...
\ No newline at end of file
+      run: go test ./...
+
+  semantic-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run semantic-release
+        # if: github.repository == 'casbin/casbin' && github.event_name == 'push'
```

Comment:
This line is necessary to skip the run semantic-release task on PR.

Reply:
@devajithvs

Reply:
Oopsie. My bad for commenting that out. I just wanted to try testing that on my repo.
I'll update that

Reply:
@sagilio plz review